code/builders/builder-webpack5/src/index.ts

```diff
@@ -114,7 +114,7 @@ export const bail: WebpackBuilder['bail'] = async () => {
 const starter: StarterFunction = async function* starterGeneratorFn({
   startTime,
   options,
-  router,
+  app,
```

Comment:
Is there any way we can make this backwards compatible?

Reply:
it may be difficult since `router` is basically the `{get, post}` shape etc

maybe we could have a wrapper around the connect server which does these 🤔 

something like:

```ts
const router = {
  get(path, callback) {
    app(path, (req, res, next) => {
      // check the method is `GET` here and do nothing if it isn't
      callback(req, res, next);
    });
  },
  // ...
};
```

although keep in mind `path` would be treated as a _prefix_ rather than a route pattern (i.e. `/foo` would capture `/foo/bar`, and there's no such thing as `/foo/:someParam` anymore, i.e. params).